packages/cli/src/ceramic-daemon.ts

```diff
@@ -36,20 +36,20 @@ import { StatusCodes } from 'http-status-codes'
 import crypto from 'crypto'
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const packageJson = JSON.parse(fs.readFileSync(new URL('../package.json', import.meta.url), 'utf8'))
+import lru from 'lru_map'
 
 const DEFAULT_HOSTNAME = '0.0.0.0'
 const DEFAULT_PORT = 7007
 const HEALTHCHECK_RETRIES = 3
 const CALLER_NAME = 'js-ceramic'
 
 const ADMIN_CODE_EXPIRATION_TIMEOUT = 1000 * 60 * 1 // 1 min
+const ADMIN_CODE_CACHE_CAPACITY = 500
```

Comment:
let's make this smaller.  Maybe 50? It's hard for me to imagine there ever being more than 5 simultaneous admin API requests in flight all at once.  Honestly its hard for me to imagine there being more than like 2 or 3.